bindings/wasm/bindings.cpp

```diff
@@ -284,19 +168,21 @@ EMSCRIPTEN_BINDINGS(whatever) {
   function("_Cube", &Manifold::Cube);
   function("_Cylinder", &Manifold::Cylinder);
   function("_Sphere", &Manifold::Sphere);
-  function("tetrahedron", &Manifold::Tetrahedron);
-  function("_Smooth", &Smooth);
-  function("_Extrude", &Extrude);
+  function("_Tetrahedron", &Manifold::Tetrahedron);
```

Comment:
intentional?

Reply:
Yes, the `class_function` method of embind was not working for me for setting up the static methods (compiles, but then the function is not actually available in `examples.js` (function undefined). Maybe because of the way that the object is being replaced in `binding.js`?

Reply:
See https://github.com/elalish/manifold/pull/440#discussion_r1207452143